libcxx/test/libcxx/ranges/range.adaptors/range.stride.view/base.nodiscard.verify.cpp

```diff
@@ -0,0 +1,28 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20
+
+// Test that std::ranges::stride_view::base() is marked nodiscard.
+
+#include <ranges>
+
```

Comment:
CI failure caused by missed inclusion.
```suggestion
#include <vector>

```

Reply:
Thanks!

Reply:
Thanks!! Fixed!